mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -106,10 +107,10 @@ def setup_and_check(self):  # {{{
         regions = [regions[index] for index in regionIndicesToPlot]
 
         for region in regions:
-            filePrefix = 'ohc_{}_{}.png'.format(region, mainRunName)
-            self.xmlFileNames.append('{}/{}.xml'.format(self.plotsDirectory,
-                                                        filePrefix))
-            self.filePrefixes[region] = filePrefix
+            for plotType in ['TZ', 'TAnomalyZ', 'SZ', 'SAnomalyZ', 'OHCZ', 'OHCAnomalyZ', 'OHC', 'OHCAnomaly']:
```

Comment:
@gstreletz and @milenaveneziani, the problem with the 4 extra XML files  is here.  We want:
```python
if plotOriginalFields:
    plotTypes = ['TZ', 'TAnomalyZ', 'SZ', 'SAnomalyZ', 'OHCZ', 'OHCAnomalyZ', 'OHC', 'OHCAnomaly']
else:
    plotTypes =  ['TAnomalyZ', 'SAnomalyZ', 'OHCAnomalyZ', 'OHCAnomaly']
...
for plotType in plotTypes:
...
```
We don't want to be adding XML file names to this list unless we're going to create them later in the code.